packages/@coorpacademy-components/src/atom/empty-state-bulk-inspect/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import style from './style.css';
+
+const EmptyStateBulkInspect = props => {
+  const {firstMessage, secondMessage} = props;
+  return (
+    <div className={style.container}>
+      <span className={style.firstMessage}>{firstMessage}</span>
+      <span className={style.secondMessage}>{secondMessage}</span>
+    </div>
+  );
+};
+
+EmptyStateBulkInspect.propTypes = {
+  firstMessage: PropTypes.string.isRequired,
+  secondMessage: PropTypes.string.isRequired
```

Comment:
un meilleur nommage : 
```suggestion
  mainText: PropTypes.string.isRequired
  subText: PropTypes.string.isRequired
```

Reply:
je me suis inspiré du composant `EmptySearchResult` (pour le nommage firstMessage, secondMessage)
<img width="995" alt="bulkImg" src="https://github.com/CoorpAcademy/components/assets/94372828/7d5fefcb-217c-4434-9ddf-2f65286c66b2">